### PR TITLE
Basic Auth is not deprecated for OPDK

### LINKED ISSUE #608: basic auth support
Hi,

We are aware that Basic Auth for Apigee Management API's (Onprem) is deprecated, yet, still available on latest apigee plugin and Drupal 9.x. 

What is the dead line where you will remove the option to work with Basic in UI UX level? 

Thanks,

### PR DESCRIPTION
**Pull request by phdhiren**
Basic Auth is not deprecated for OPDK, removed deprecated message for private cloud.
Closes #608

### CODE REVIEW

File: src/Plugin/KeyInput/ApigeeAuthKeyInput.php [hunk 1]
```diff
@@ -89,6 +89,9 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
         'required' => [$state_for_public, $state_for_private],
       ],
     ];
+    if (EdgeKeyTypeInterface::INSTANCE_TYPE_PRIVATE === $values['instance_type']) {
```
Comment (shishir-intelli) [inline src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: @phdhiren I think we should check `!empty` for `instance_type` here also.

File: js/apigee_edge.admin.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   *
+   * @prop {Drupal~behaviorAttach} attach
+   */
+  Drupal.behaviors.apigeeEdgeAuthForm = {
+    attach: function attach(context, drupalSettings) {
+      $('#edit-key-input-settings-instance-type input:radio').click(function() {
+        if ($(this).val() === 'private') {
+          $('#edit-key-input-settings-auth-type option[value="basic"]').text(Drupal.t('HTTP basic'));
+        } else if ($(this).val() === 'public') {
+          $('#edit-key-input-settings-auth-type option[value="basic"]').text(Drupal.t('HTTP basic (deprecated))');
```
Comment (shishir-intelli) [inline js/apigee_edge.admin.js]: @phdhiren Somehow I feel there is something wrong in this line `HTTP basic (deprecated))` seems syntax error.

**Review by shishir-intelli — COMMENTED** [review]
Syntax Error - missing ) after argument list

File: src/Plugin/KeyInput/ApigeeAuthKeyInput.php (hunk 1 above)
Comment (phdhiren) [reply src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: @shishir-intelli, thanks. Now this also covered

File: js/apigee_edge.admin.js (hunk 2 above)
Comment (phdhiren) [reply js/apigee_edge.admin.js]: @shishir-intelli This is also taken care.

**Review by shishir-intelli — APPROVED** [review]
LGTM

**Pull request merged**